s3prl/downstream/asr/hf_hub_templates/README.md

```diff
@@ -0,0 +1,15 @@
+---
+library_name: superb
```

Comment:
this yaml metadata is used for filtering / automatic widget creation on the hugging face hub